environments/environments.py

```diff
@@ -102,21 +94,26 @@ def sample_random_rollouts(self,
             img_size (int): Size of the frames (in pixels)
             color (bool): Whether to have colored or grayscale frames.
             noise_std (float): If noisy_data=true, standard deviation of the gaussian noise source.
-            base_radius (float): Init phase states will be sampled from a circle (q, p) of radius 
-                r ~ U(base_radius, base_radius + 1) https://arxiv.org/pdf/1909.13789.pdf (Sec. 4)
-            seed (int): Seed for reproducibility
-
+            radius_lb (float): Radius lower bound of the phase state sampling.
```

Comment:
maybe a tuple is better rather than 2 parameters?

Reply:
tuple better!